infra/modular-examples/aws/main.tf

```diff
@@ -34,6 +34,7 @@ module "worklytics_connector_specs" {
   google_workspace_example_user  = var.google_workspace_example_user
   google_workspace_example_admin = try(coalesce(var.google_workspace_example_admin, var.google_workspace_example_user), null)
   salesforce_domain              = var.salesforce_domain
+  msft_tenant_id                 = var.msft_tenant_id
```

Comment:
this is the fix, pass through `msft_tenant_id` to `worklytics_connector_specs` module